bool is_fix  # true to fix, false to unfix
geometry_msgs/PoseStamped pose # pose to find the nearest fabric particle